addons/bus/websocket.py

```diff
@@ -572,7 +572,13 @@ def _handle_transport_error(self, exc):
             code = CloseCode.SESSION_EXPIRED
         if code is CloseCode.SERVER_ERROR:
             reason = None
-            _logger.error(exc, exc_info=True)
+            registry = Registry(self._session.db)
+            sequence = registry.registry_sequence
+            registry = registry.check_signaling()
+            if sequence != registry.registry_sequence:
+                _logger.warning("Bus operation aborted; registry has been reloaded")
+            else:
+                _logger.error(exc, exc_info=True)
```

Comment:
If the registry sequence changes during check_signaling, the returned registry will change too

```suggestion
            registry = Registry(self._session.db)
            new_registry = registry.check_signaling()
            if registry !=new_registry :
                _logger.warning("Bus operation aborted; registry has been reloaded")
            else:
                _logger.error(exc, exc_info=True)
```

Reply:
I find it more explicit to use the sequence